src/ostree.cc

```diff
@@ -23,6 +23,32 @@ OstreeSysroot *Ostree::LoadSysroot(const std::string &path) {
   return sysroot;
 }
 
+OstreeDeployment *Ostree::getStagedDeployment(const std::string &path, const std::string &ostree_os) {
+  OstreeSysroot *sysroot = Ostree::LoadSysroot(path);
+  GPtrArray *deployments = NULL;
+  OstreeDeployment *res = NULL;
+
+  const char *osname;
+  if (ostree_os.empty()) {
+    OstreeDeployment *merge_deployment = ostree_sysroot_get_merge_deployment(sysroot, NULL);
```

Comment:
Are You sure that merge_deployment is deployment that will boot after reboot? What will happen if we install two packages one by one than reboot, will merge_deployment point to latest one? Will it point after reboot to current deployment?